### PR TITLE
Adding scram-sha-256 for postgres >=10

### PR DESCRIPTION
**Pull request by fao89**
Fixing lint + https://github.com/geerlingguy/ansible-role-postgresql/pull/162

### CODE REVIEW

**Comment by fao89** [comment]
@mikedep333

**Comment by bmbouter** [comment]
How can I test this? I have a pulp_installer checkout, but I don't know how to force it to use your branch instead of the real one from galaxy.ansible.com.

**Comment by mikedep333** [comment]
If you do not install it as a

> How can I test this? I have a pulp_installer checkout, but I don't know how to force it to use your branch instead of the real one from galaxy.ansible.com.

I'm fairly certain this will work:
```
diff --git a/requirements.yml b/requirements.yml
index dff95e2..59bf938 100644
--- a/requirements.yml
+++ b/requirements.yml
@@ -1,5 +1,7 @@
 ---
 roles:
-  - geerlingguy.postgresql
+  - name: geerlingguy.postgresql
+    src: https://github.com/fao89/ansible-role-postgresql.git
+    version: lint_scram
 collections:
   - ansible.posix
```

File: tasks/users.yml [hunk 1]
```diff
@@ -2,10 +2,13 @@
 - name: Ensure PostgreSQL users are present.
   postgresql_user:
     name: "{{ item.name }}"
+    password: "{{ item.password | default(omit) }}"
```
Comment (fao89) [inline tasks/users.yml:5]: Currently, `community.postgresql` cannot compare `scram-sha-256 ` with `plain text` or `md5`
https://github.com/ansible-collections/community.postgresql/blob/main/plugins/modules/postgresql_user.py#L387-L389

**Comment by fao89** [comment]
I'll close it in favor of https://github.com/geerlingguy/ansible-role-postgresql/pull/163
@mikedep333 @bmbouter

**Pull request closed without merging**